eng/common/docgeneration/Generate-DocIndex.ps1

```diff
@@ -188,5 +188,6 @@ if ($GetGithubIoDocIndexFn -and (Test-Path "function:$GetGithubIoDocIndexFn"))
 }
 else
 {
-    LogWarning "The function 'GetGithubIoDocIndexFn' was not found."
+    LogWarning "The function 'GetGithubIoDocIndexFn' was not found.`
```

Comment:
@danieljurek this update is regards to your feedback. What information do you think would have helped you fixing the cpp issue you hit?